eng/common/templates/job/source-index-stage1.yml

```diff
@@ -1,6 +1,6 @@
 parameters:
   runAsPublic: false
-  sourceIndexPackageVersion: 1.0.1-20231213.4
+  sourceIndexPackageVersion: 1.0.1-20240129.1
```

Comment:
@eerhardt this change should also be made in arcade - this file is in `eng/common`, so any changes here will be overwritten by the next arcade update

Reply:
Good call. Will send an arcade PR.